provisioning/service/src/Config/IndividualEnrollment.cs

```diff
@@ -153,12 +154,19 @@ public IndividualEnrollment(string registrationId, Attestation attestation)
             DateTime lastUpdatedDateTimeUtc,
             string eTag)
         {
+            /* SRS_INDIVIDUAL_ENROLLMENT_21_003: [The constructor shall throws ProvisioningServiceClientException if one of the 
+                                                    provided parameters in JSON is not valid.] */
             if (string.IsNullOrWhiteSpace(eTag))
             {
                 throw new ProvisioningServiceClientException("Service respond an individualEnrollment without eTag.");
             }
+            if (attestation == null)
+            {
+                throw new ProvisioningServiceClientException("Service respond an individualEnrollment without attestation.");
```

Comment:
Same as above. Code should always throw `ArgumentNullException` when a required argument is `null`.
I believe this may fail FXCop as well.

Reply:
Same as above, this null is because the service sent a wrong answer.